src/main/java/com/monumental/models/api/MonumentAboutPageStatistics.java

```diff
@@ -0,0 +1,91 @@
+package com.monumental.models.api;
+
+import com.monumental.models.Monument;
+
+import java.util.HashMap;
+
+/**
+ * Model class that holds statistics about the Monuments in the system for the About Page
+ */
+public class MonumentAboutPageStatistics {
```

Comment:
I don't think this necessarily belongs in the model package if it doesn't extend `Model`, although I'm not sure what the best place for it is

Reply:
I could move them into a subfolder of `com.monument.controllers`, something like `helpers` or something. Thoughts?

Reply:
Fine by me

Reply:
Done.